### PR TITLE
Tune-up of VCS documentation

### PR DESCRIPTION
**Pull request by comradekingu**
Should it be "SVN co" instead of "svn co"? Not sure what that is.

### CODE REVIEW

File: docs/vcs.rst [hunk 1]
```diff
@@ -26,16 +26,16 @@ Accessing repositories from Hosted Weblate
 For Hosted Weblate there is a dedicated push user registered on GitHub,
 Bitbucket, Codeberg and GitLab (with username :guilabel:`weblate` named
 :guilabel:`Weblate push user`). You need to add this user as a collaborator and
-give it appropriate permission to your repository (read only is okay for
+give it appropriate permission to your repository (read-only is okay for
 cloning, write is required for pushing). Depending on service and your
-organization settings, this happens immediately or requires confirmation from
+organization settings, this happens immediately, or requires confirmation on
```
Comment (comradekingu) [inline docs/vcs.rst:31]:
```suggestion
organization settings, this happens immediately, or requires confirmation on the
```

File: docs/vcs.rst [hunk 2]
```diff
@@ -130,14 +130,17 @@ approach is also used for Hosted Weblate, there is dedicated
 Weblate internal URLs
 +++++++++++++++++++++
 
-To share one repository between different components you can use a special URL
-like ``weblate://project/component``. This way, the component will share the
-VCS repository configuration with the referenced component
-(``project/component`` in the example).
+Share one repository setup between different components by reffering to
+its placement as ``weblate://project/component``. This way other components
```
Comment (comradekingu) [inline docs/vcs.rst:134]:
```suggestion
its placement as ``weblate://project/component`` in other components. This way other components
```

File: docs/vcs.rst [hunk 3]
```diff
@@ -49,7 +49,7 @@ repository this way.
 
 .. warning::
 
-    On GitHub, each key can be use only once, see :ref:`vcs-repos-github` and
+    On GitHub, each key can only be used once, see :ref:`vcs-repos-github` and
     :ref:`hosted-push`.
 
 Weblate also stores the host key fingerprint upon first connection, and fails to
```
Comment (comradekingu) [inline docs/vcs.rst:55]:
```suggestion
Weblate also stores the host-key fingerprint upon first connection, and fails to
```

Comment (comradekingu) [inline docs/vcs.rst:55]:
```suggestion
Weblate also stores the fingerprint for the host key upon first connection, and fails to
```

File: docs/vcs.rst [hunk 4]
```diff
@@ -130,14 +130,17 @@ approach is also used for Hosted Weblate, there is dedicated
 Weblate internal URLs
 +++++++++++++++++++++
 
-To share one repository between different components you can use a special URL
-like ``weblate://project/component``. This way, the component will share the
-VCS repository configuration with the referenced component
-(``project/component`` in the example).
+Share one repository setup between different components by reffering to
```
Comment (comradekingu) [inline docs/vcs.rst:133]:
```suggestion
Share one repository setup between different components by referring to
```

**Comment by nijel** [comment]
`svn` is name of Subversion command, that certainly should not be changed ;-)

**Review by orangesunny — APPROVED** [review]
good, thanks for the contribution, @comradekingu! Let’s merge.

**Pull request merged**